src/cmd/services/m3aggregator/config/aggregator.go

```diff
@@ -119,6 +122,12 @@ type AggregatorConfiguration struct {
 	// Flushing handler configuration.
 	Flush handler.FlushHandlerConfiguration `yaml:"flush"`
 
+	// PassThrough m3msg topic name.
+	PassThroughTopicName *string `yaml:"passThroughTopicName"`
+
+	// The number of passthrough writers
+	NumPassThroughWriters *int `yaml:"numPassThroughWriters"`
```

Comment:
Do these need to be nullable? Can we just use defaults + config population?

Reply:
sure